web_widget_json_graph/__manifest__.py

```diff
@@ -1,22 +1,25 @@
 {
     "name": "Web Widget JSON Graph",
-    "version": "12.0.1.0.0",
+    "version": "15.0.1.0.1",
     "author": "Vauxoo, Odoo Community Association (OCA)",
     "license": "LGPL-3",
     "category": "Hidden/Dependency",
     "website": "https://github.com/OCA/web",
-    "maintainers": ["luisg123v"],
+    "maintainers": ["frahikLV"],
     "summary": "Draw json fields with graphs.",
     "depends": [
         "web",
     ],
-    "data": [
-        "views/assets.xml",
-    ],
-    "qweb": [
-        "static/src/xml/web_widget_json_graph.xml",
-    ],
-    "installable": True,
+    "data": [],
+    "assets": {
+        "web.assets_backend": [
+            "web_widget_json_graph/static/src/js/web_widget_json_graph.js"
```

Comment:
Can you add the comma at the end?? So if more assets are added later on just a line is added and they don't have to edit your line to add the comma and add another line, also it helps keep a better track of commits 😄 

Same in the "web.assets_qweb"

Reply:
Done